### DISCUSSION TITLE
KDE Plugin

### DISCUSSION THREAD

**Discussion by BlackHawk185** [Almamu/linux-wallpaperengine#65] [Ideas]
So the folks over at KDE Plasma recently got Wallpaper engine running basically flawlessly. I was wondering what the key differences in architecture between KDE and GDM are, and if this could potentially boost the GDM version in any way.

**Comment by Almamu** [comment]
As far as I know the KDE versión that you're talking about runs as a KDE plugin and is only compatible with KDE. This project aims to offer a version that works on as much DEs as possible, but there's still work to do on the renderer before actually being able to implement any actual "running in the background" for complex DEs.
